slices/gearman-job-server.yaml

```diff
@@ -0,0 +1,28 @@
+package: gearman-job-server
+
+essential:
+  - gearman-job-server_copyright
+
+slices:
+  libs:
+    essential:
```

Comment:
Can you include a comment as follow explaining why `adduser` is being omitted as a dependency?
```
    # gearman-job-server actually depends on adduser as well to create and
    # remove a user during package install/removal - however we don't support
    # this currently so adduser is not added here.
```